server/app/services/member-service.js

```diff
@@ -205,6 +207,30 @@ async function deleteMember(memberId) {
   return member;
 }
 
+/**
+ * Search members which match query parameter in all database.
+ *
+ * @param query {string} query.
+ * @return {Promise<Member[]>} Members which match the query
+ */
+async function searchMembers(query) {
+  logger.verbose('Member service: searching members with query %s', query);
+  const where = Sequelize.where(
+    Sequelize.fn('concat', Sequelize.col('firstName'), ' ', Sequelize.col('lastName')),
+    'like',
+    `%${query}%`,
+  );
+  const members = await Member.findAll({
+    where,
+    include: [{
+      model: Registration,
+      as: 'registrations',
+      attributes: ['year', 'createdAt'],
```

Comment:
An order by maybe? Also I discovered `scopes` in sequelize and it looks really nice! http://docs.sequelizejs.com/manual/tutorial/scopes.html